wire-grpc-tests/src/test/proto-grpc/routeguide/Feature.kt

```diff
@@ -70,6 +70,13 @@ data class Feature(
     return result
   }
 
+  override fun toString(): String = buildString {
+    append("Feature{")
+    if (name != null) append("""name=$name""")
+    if (location != null) append(""", location=$location""")
```

Comment:
if `name` is null and `location` is not the output is
```
Feature{, location=...}
```

Here's a workaround I've used in the past:
```kotlin
append("Featur") // <-- should be name minus one character
if (name != null) append(""", name=$name""")
if (location != null) append(""", location=$location""")
set(6, 'e') // these values change based on the length of the name
set(7, '{')
append('}')
```
Since the shortest name is of size 1, this always works. (And in that case you'd have no initial `append`)

Reply:
Oh dear it looks like mutating functions on StringBuilder are non-existent in common code 😭

Reply:
Ugh.. Will fix, good catch.

Reply:
could do a `listOf().joinAsString` as well right?

Reply:
You would still need to filter nulls and now you're talking about multiple collection allocations, the array allocation to the varargs, their internal expansion when more than 10 properties are defined, etc.

Not great, but might be fine to start with.